src/gen-ts.ts

```diff
@@ -132,6 +132,9 @@ ${sourceUrls.map((url) => '  ' + url).join('\n')}`;
  */
 function handleDocument(doc: analyzer.Document, root: ts.Document) {
   for (const feature of doc.getFeatures()) {
+    if ((feature as any).privacy === 'private') {
```

Comment:
This is slightly gross, but it should work.

Reply:
Yeah, I guess I could add a return in every handle method instead?

Reply:
We'll probably have a lower defect rate with it here because we won't forget anywhere it needs to go. Features will never be undefined, so this should be fine.

Hm, actually what about:

```typescript
interface MaybePrivacy {
  privacy?: Privacy;
}

const privacyFeature: MaybePrivacy = feature;
if (privacyFeature.privacy === 'private') {
  // ...
}
```

Reply:
That doesn't really add any type safety though, just sort of gives the illusion of it?

Reply:
Not so! https://www.typescriptlang.org/play/#src=type%20Privacy%20%3D%20'private'%20%7C%20'public'%3B%0D%0Ainterface%20MaybePrivate%20%7B%0D%0A%20%20%20%20privacy%3F%3A%20Privacy%3B%0D%0A%7D%0D%0A%0D%0Alet%20maybePrivate%3A%20MaybePrivate%3B%0D%0AmaybePrivate%20%3D%20%7B%7D%3B%0D%0AmaybePrivate%20%3D%20%7B%20privacy%3A%20'public'%20%7D%3B%0D%0AmaybePrivate%20%3D%20null%3B%0D%0AmaybePrivate%20%3D%20%7B%20privacy%3A%20%7B%20alot%3A%20'yes'%20%7D%20%7D%3B%0D%0A

Reply:
Oh, good point! It enforces that if there *is* a thing called`privacy`, it looks like the way we think. Done.